packages/block-editor/src/components/block-alignment-control/ui.js

```diff
@@ -39,7 +55,8 @@ const BLOCK_ALIGNMENTS_CONTROLS = {
 	},
 };
 
-const DEFAULT_CONTROL = 'center';
+const DEFAULT_CONTROL = 'none';
+const help = __( 'Wider alignments are not available.' );
```

Comment:
I actually think this message is not helpful at all, because it can confuse the user in assuming that wide alignments are not available anywhere while they are container-dependent.

Reply:
The `copy` of this is not finalized: https://github.com/WordPress/gutenberg/issues/34597#issuecomment-913925613 and IMO we could definitely could omit that here and add it in a follow up..

Reply:
I'm in the opinion that we shouldn't add it at all but 🤷‍♂️

Reply:
This is for cases like within a column where wide doesn't make sense? Let's skip showing anything there. We should only show a message on container where wide/full could be supported but are not.

Reply:
@mtias is it okay to skip in this PR? I think it needs more thinking and could be added in a follow up - especially if we try to add some logic to find if a block is a container or not.

Reply:
Sounds good